### PR TITLE
Update prereqs info

### PR DESCRIPTION
**Pull request by paigehargrave**
@federicoboerr - PTAL - these changes are for https://www.mulesoft.org/jira/browse/DOCS-4922 (and subsequent discussion in slack channel). Thank you!

### CODE REVIEW

File: modules/ROOT/pages/prereq-software.adoc [hunk 1]
```diff
@@ -8,12 +8,14 @@ The following sections list the software required to install and run the Anypoin
 == Supported Linux Distributions
 The following Linux distributions are supported:
 
-* Red Hat Enterprise Linux (RHEL), versions 7.4.x, and 7.5.x
+* Red Hat Enterprise Linux (RHEL), versions 7.4.x, 7.5.x, and 7.6.x
 
 * CentOS, versions 7.4.x, and 7.5.x
```
Comment (federicoboerr) [inline modules/ROOT/pages/prereq-software.adoc]: `versions 7.4.x, 7.5.x, and 7.6.x` also applies here :)

Comment (alejandrorascovan) [inline modules/ROOT/pages/prereq-software.adoc]: 7.6.x?

Comment (paigehargrave) [reply modules/ROOT/pages/prereq-software.adoc]: Added - thank you!

Comment (paigehargrave) [reply modules/ROOT/pages/prereq-software.adoc]: Removed the '.x' to avoid confusion. Thank you!

**Comment by paigehargrave** [comment]
alejandrorascovan - Does this look OK now?

File: modules/ROOT/pages/prereq-network.adoc [hunk 2]
```diff
@@ -39,7 +39,10 @@ The version of Kubernetes supported by Anypoint Platform Private Cloud Edition u
 
 == NAT Traffic Requirements
 
-In some situation, the Kubernetes overlay network uses NAT. NAT requires that servers be able to send and receive packages with a source and destination that is different from server’s internal IP.
+In some situation, the Kubernetes overlay network uses NAT. NAT requires that servers be able to send and
```
Comment (valkyrie69) [inline modules/ROOT/pages/prereq-network.adoc:46]: situation**s**
Line 43 ...different from the server's internal IP.

File: modules/ROOT/pages/prereq-network.adoc [hunk 3]
```diff
@@ -63,6 +66,8 @@ To ensure proper cluster load balancing and routing, you must enable and correct
 * Kernel IP forwarding: to enable internal Kubernetes load balancing you must enable IPv4 forwarding on all servers.
 * Bridge-netfilter: enables the Linux kernel to translate packets to and from hosted containers.
 
+** IP addresses for bridge-netfiltering are managed by the overlay network created by kubernetes. The CIDR block used for that network is in the private IP address range.
```
Comment (valkyrie69) [inline modules/ROOT/pages/prereq-network.adoc]: Kubernetes

File: modules/ROOT/pages/install-workflow.adoc [hunk 4]
```diff
@@ -13,6 +13,10 @@ The installer is interactive and walks you through a set of wizard pages that en
 
 The CLI installer does not require human interaction using a web browser and is useful for automation.
 
+NOTE: The installation package includes everything required for Anypoint Private Cloud Edition installation,
```
Comment (valkyrie69) [inline modules/ROOT/pages/install-workflow.adoc]: Format as a note if it's a note. 
[NOTE]
blah blahblahblah

**Review by valkyrie69 — APPROVED** [review]
Minor comments

File: modules/ROOT/pages/install-workflow.adoc (hunk 4 above)
Comment (paigehargrave) [reply modules/ROOT/pages/install-workflow.adoc]: OK - thank you!

File: modules/ROOT/pages/prereq-network.adoc (hunk 2 above)
Comment (paigehargrave) [reply modules/ROOT/pages/prereq-network.adoc:46]: Good catch - thank you!

File: modules/ROOT/pages/prereq-network.adoc (hunk 3 above)
Comment (paigehargrave) [reply modules/ROOT/pages/prereq-network.adoc]: Got it - thank you!

**Pull request merged**